docs/pages/component/props.mdx

```diff
@@ -343,6 +343,8 @@ Controls the iOS silent switch behavior
 
 <PlatformsList types={['iOS']} />
 
+**This is deprecated, prop moved to `drm`**
```

Comment:
can you pleas just copy this section in the DRM documentation ?

Reply:
Done

Reply:
👍